posthog/warehouse/api/managed_viewset.py

```diff
@@ -0,0 +1,127 @@
+import structlog
+from rest_framework import serializers, status, viewsets
+from rest_framework.request import Request
+from rest_framework.response import Response
+
+from posthog.api.routing import TeamAndOrgViewSetMixin
+from posthog.warehouse.models import ManagedViewSet
+
+logger = structlog.get_logger(__name__)
+
+
+class ManagedViewSetSerializer(serializers.Serializer):
+    enabled = serializers.BooleanField(required=True)
+
+
+class ManagedViewSetViewSet(TeamAndOrgViewSetMixin, viewsets.ViewSet):
+    scope_object = "INTERNAL"
+    lookup_field = "kind"
+    lookup_url_kwarg = "kind"
```

Comment:
Should this not have:
```
queryset = ManagedViewSet.objects.all()
```
?

Then you won't need to do the `team_id` filter everywhere